tempest-internal/src/main/kotlin/app/cash/tempest/internal/Codec.kt

```diff
@@ -220,16 +224,18 @@ private sealed class Binding<A, D, P> {
   abstract val appProperty: KProperty1<A, P>
   abstract val dbProperties: List<KProperty1<D, P>>
 
-  fun getApp(value: A) = appProperty.get(value)
+  fun getApp(value: A): P {
+    if (!appProperty.isAccessible) {
+      appProperty.javaField!!.trySetAccessible()
```

Comment:
would it be better to set these as accessible when you init the appProperty field?